andela/valinor-ah-frontend
# #162606962 Serve landing page

#### What does this PR do?
Serve the landing page

#### Description of Task to be completed?
- set up actions and reducer to handle fetching article by category
- ensure landing page loads data from the backend

#### How should this be manually tested?
- check out to this branch `feature/162606962/serve-landing-page`
- run `npm start` to start up the server
- open `localhost:3000` to view the landing page
- use redux dev tool to observe the state and state history

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162606962](https://www.pivotaltracker.com/story/show/162606962)

#### Screenshots (if appropriate)

#### Questions:
N/A

#### Link to Review app:
[review app](https://valinor-ah-frontend-stag-pr-26.herokuapp.com/)
